README.md

```diff
@@ -1,110 +1,136 @@
-  storage.
-- Recording rules aren't supported.
-- Alerts aren't supported.
-- When sharding the Agent, if your node has problems that interrupt metric
-  availability, metrics tracking that node won't be sent for alerting on.
-
-While the Agent can't use recording rules and alerts, `remote_write` systems such
-as Mimir currently support server-side rules and alerts. Note that this trade-off
-means that reliability of alerts are tied to the reliability of the remote system
-and alerts will be delayed at least by the time it takes for samples to reach
-the remote system.
+Grafana Agent is a vendor-neutral, batteries-included telemetry collector with
+configuration inspired by [Terraform][]. It is designed to be flexible,
+performant, and compatible with multiple ecosystems such as Prometheus and
+OpenTelemetry.
+
+Grafana Agent is based around **components**. Components are wired together to
+form programmable observability **pipelines** for telemetry collection,
+processing, and delivery.
+
+> **NOTE**: This page focuses mainly on "[Flow mode][Grafana Agent Flow]," the
+> Terraform-inspired revision of Grafana Agent.
+
+Grafana Agent can collect, transform, and send data to:
+
+* The [Prometheus][] ecosystem
+* The [OpenTelemetry][] ecosystem
+* The Grafana open source ecosystem ([Loki][], [Grafana][], [Tempo][], [Mimir][], [Phlare][])
+
+[Terraform]: https://terraform.io
+[Grafana Agent Flow]: https://grafana.com/docs/agent/latest/flow/
+[Prometheus]: https://prometheus.io
+[OpenTelemetry]: https://opentelemetry.io
+[Loki]: https://github.com/grafana/loki
+[Grafana]: https://github.com/grafana/grafana
+[Tempo]: https://github.com/grafana/tempo
+[Mimir]: https://github.com/grafana/mimir
+[Phlare]: https://github.com/grafana/phlare
+
+## Why use Grafana Agent?
+
+* **Vendor-neutral**: Fully compatible with the Prometheus, OpenTelemetry, and
+  Grafana open source ecosystems.
+* **Every signal**: Collect telemetry data for metrics, logs, traces, and
+  continuous profiles.
+* **Scalable**: Deploy on any number of machines to collect millions of active
+  series and terabytes of logs.
+* **Battle-tested**: Grafana Agent extends the existing battle-tested code from
+  the Prometheus and OpenTelemetry Collector projects.
+* **Powerful**: Write programmable pipelines with ease, and debug them using a
+  [built-in UI][UI].
+* **Batteries included**: Integrate with systems like MySQL, Kubernetes, and
+  Apache to get telemetry that's immediately useful.
+
+[UI]: https://grafana.com/docs/agent/latest/flow/monitoring/debugging/#grafana-agent-flow-ui
 
 ## Getting started
 
-When using Kubernetes this [link](https://grafana.com/docs/grafana-cloud/quickstart/agent-k8s) offers the best guide.
+Check out our [documentation][] to see:
 
-Other installation methods can be found in our
-[Grafana Agent](https://grafana.com/docs/agent/latest/set-up/) documentation.
+* [Installation instructions][] for Grafana Agent
```

Comment:
All the installation instructions here point to static mode configuration which might be confusing. Can we start thinking about changing those as well?

Reply:
Yeah, definitely. We need to start thinking in general what we want to do with the documentation, and whether it makes sense to create a dedicated static mode folder.